salesforce/10.10/modules/ROOT/pages/salesforce-connector-config-topics.adoc

```diff
@@ -45,27 +47,50 @@ output application/java
             </ee:message>
 </ee:transform>
 ----
+== Update Field Values to Null
 
+To set field values to null, use an *Update* or *Upsert* request operation and set the name of the fields you want to nullify in the *Transform Message* component's *fieldsToNull* field:
 
-=== Message Redelivery for Input Sources
-
-You can configure a redelivery policy for input sources by setting the number of redelivery attempts to try after an initial failure. You can specify an object store for this policy. If you don't specify an object store, Mule creates a non-persistent object store.
-
-The number of transactions used to interact with the object store varies based on the number of retries configured for the redelivery policy.
-
-For more information about configuring a redelivery policy, see xref:mule-runtime::redelivery-policy.adoc[Redelivery Policy].
-
-=== OAuth 2.0 Connection Type
+.Specify the field names you want to nullify in the *Transform Message* component *fieldsToNull* field.
+image::salesforce-connector-10-set-fields-to-null.png[Set a field to null]
 
-When configuring an OAuth 2.0 connection, you can specify an object store that stores each resource owner's ID data. If you don't specify an object store, Mule automatically provisions the default object store.
+== Generate a Keystore File
 
-The app interacts with the object store automatically when a new resource owner is authenticated, the access token is refreshed, or the access token is invalidated.
+Salesforce Connector uses the Bouncy Castle cryptographic library to load the certificate from the keystore and sign the payload for the authentication requests.
 
-== Clear Fields by Updating Field Value to Null
+The library has a https://nvd.nist.gov/vuln/detail/CVE-2018-5382[vulnerability (CVE-2018-5382)] that relates to the BKS-V1 keystore file type. By default, Salesforce Connector uses JKS or PKCS12 keystores. It is best not to use BKS-V1 keystore files, as documented in the https://www.kb.cert.org/vuls/id/306792[Bouncy Castle proposed solution].
 
-In Salesforce Connector 10.x, to clear a field, you must use an *Update* or *Upsert* request operation and set the name of the fields you want to nullify in the *fieldsToNull* field, for example:
+To generate a JKS keystore file:
 
-image::salesforce-connector-10-set-fields-to-null.png[Set a field to null ]
+. Go to your Mule workspace and open the command prompt (for Windows) or Terminal (for Mac).
+. Type this command and press enter:
```

Comment:
"Enter this command"?
Avoids the verb contention (press? tap? etc.), the name of the key on the keyboard, and whether or not to capitalize the key name.